content-security-policy/support/testharness-helper.sub.js

```diff
@@ -5,8 +5,30 @@ const Host = {
 
 const PolicyHeader = {
   CSP: "echo-policy.py?policy=",
+  CSP_MULTIPLE: "echo-policy-multiple.py",
   EMBEDDING_CSP: "echo-embedding-csp.py",
-} 
+  ALLOW_CSP_FROM: "echo-allow-csp-from.py",
+};
+
+const IframeLoad = {
+  EXPECT_BLOCK: true,
+  EXPECT_LOAD: false,
+};
+
+function getOrigin() {
+  var url = new URL("http://{{host}}:{{ports[http][0]}}/");
+  return url.toString();
+}
+
+function getCrossOrigin() {
+  var url = new URL("http://{{domains[天気の良い日]}}:{{ports[http][0]}}/");
+  return url.toString();
+}
+
+function getSecureCrossOrigin() {
+  var url = new URL("https://{{domains[天気の良い日]}}:{{ports[http][0]}}/");
```

Comment:
It seems like server is that such that ` {{ports[https][0]}}` is 8443, whereas ` {{ports[http][0]}}` is 8000.
So that `'self'` is http://xn--n8j6ds53lwwkrqhv28a.web-platform.test:8000/
and secure server https give `https://xn--n8j6ds53lwwkrqhv28a.web-platform.test:8443/`

I just wanted to add checks for 'self' that are not exact matches (for example, more secure scheme). But since ports differ and 8443 is not default for `https`, subsumption is false. So I wanted to use more secure scheme but the same port. For now I added a comment, but we could also just drop this test?